compiler/main.nim

```diff
@@ -342,8 +342,6 @@ proc mainCommand*(graph: ModuleGraph) =
     if conf.projectIsCmd or conf.projectIsStdin: discard
     elif not fileExists(conf.projectFull):
       rawMessage(conf, errGenerated, "NimScript file does not exist: " & conf.projectFull.string)
-    elif not conf.projectFull.string.endsWith(".nims"):
```

Comment:
this doesn't work:
`nim e foo.nimfake` doesn't seem to run nimscript on foo.nimfake.

also, ideally this would work too: `nim r main.nims`, but can be done later

Reply:
nim e foo.nimfake

what's the result?

Reply:
no error message, but nothing runs (eg if it contains `echo "foo"`, it won't print foo)

Reply:
i see